index.html

```diff
@@ -5,13 +5,45 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <meta name="description" content="This is a knowledge based website where you get to ask any question and get the answers you need" />
+    <meta name="description"
+        content="This is a knowledge based website where you get to ask any question and get the answers you need" />
     <meta name="keywords" content="questions,search, comments, opinions" />
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Quicksand&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Luckiest+Guy&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Monoton&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Aclonica&display=swap" rel="stylesheet">
     <title>Knowledge Based Homepage</title>
-    <link rel="stylesheet" href="css/style.css">
+
 </head>
 
 <body>
+    <header>
+        <nav id="navbar">
+            <div class="container">
+                <h1 class="logo" a href="#">Knowledge Base</h1>
+                <ul>
+                    <li><a href="#">Home</a></li>
+                </ul>
+
+            </div>
+
+
+        </nav>
+    </header>
+
+
+    <main class="py-3">
+        <div id="main-header">
+            <h3 id="main-header__1"> Have Any Question to Ask?</h3>
+            <h5 id="main-header__2">We Have Answers to All Your Questions</h5>
```

Comment:
naming convention here isi not good enough...think or using` main-header__primary` or `main-header__secondary`